rasa/core/tracker_store.py

```diff
@@ -127,7 +127,7 @@ def load_tracker_from_module_string(
 
         if custom_tracker:
             return custom_tracker(
-                domain=domain, url=store.url, event_broker=event_broker, **store.kwargs
+                domain=domain, host=store.url, event_broker=event_broker, **store.kwargs
```

Comment:
how about using `arguments_of` (from `common`) to make it non-breaking?

Reply:
+ deprecating `host` maybe

Reply:
so in that case (deprecating host) it would be an `arguments_of` in each of the existing classes yeah? with a deprecation message

Reply:
> in each of the existing classes 

only in the custom tracker store, right? Why in the others as well?

Reply:
The changes I've implemented here deprecates `url` which would be in the custom tracker store. Deprecating `host` I think might make more sense, for the reasons I mentioned above (every other param matches those in endpoints.yml, where it's url), which would have to be done in all the classes but custom

Reply:
Or we do it in the other way around 😄  Both url and host are sometimes wrong and sometimes the right names depending on the tracker store. But actually using `arguments of` and deprecating things, would be great 🚀

Reply:
> Both url and host are sometimes wrong and sometimes the right names depending on the tracker store 

Then does it make more sense to let the store itself decide? E.g. get the relevant param from the endpoint config:
```suggestion
                domain=domain, event_broker=event_broker, **store.kwargs
```

Reply:
Mhm, or just support both?